src/lib/db.ts

```diff
@@ -87,6 +87,8 @@ export interface OpenDocumentOptions {
   groupUserConnections?: Record<string, unknown>;
   originDoc?: string;
   pubVersion?: number;
+  problemOrdinal?: string;
```

Comment:
After reviewing this I'd suggest going back to having a separate problem and investigation integers both here and in the DocumentModel. That will make it consistent with the metadata and make it easier to filter documents in the documents store.

Reply:
OK, I've done that in the latest commit.